### PR TITLE
Fix #3551 option audio language.xml

### LINKED ISSUE #3551: Merge option_audio_language.xml into single xml file
Currently there are 3 versions of `option_audio_language.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by shankarpriyank**
PTAL @bkaur-bkj
## Explanation
- Merged 2 files of option_audio_language.xml
  - "Fixes #3551:" 

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing
- oopia landscape
oppia portrait
oppia tablet  landscape
oppia tablet potrait
rtl oppia
rtl oppia1

### CODE REVIEW

**Review by bkaur-bkj — COMMENTED** [review]
@shankarpriyank merging of mobile layout files looks fine to me, have you checked for the tablet layout files?

**Comment by shankarpriyank** [comment]
> @shankarpriyank merging of mobile layout files looks fine to me, have you checked for the tablet layout files?
It has some differences I don't know how to deal with , which are
` <variable
      name="commonViewModel"
      type="org.oppia.android.app.options.OptionControlsViewModel" />
 android:background="@{commonViewModel.selectedFragmentIndex == viewModel.itemIndex ? @drawable/general_item_background_border_cyan : @drawable/general_item_background_border}"`

**Comment by bkaur-bkj** [comment]
> > @shankarpriyank merging of mobile layout files looks fine to me, have you checked for the tablet layout files?
> > It has some differences I don't know how to deal with , which are
> > ` <variable       name="commonViewModel"       type="org.oppia.android.app.options.OptionControlsViewModel" />  android:background="@{commonViewModel.selectedFragmentIndex == viewModel.itemIndex ? @drawable/general_item_background_border_cyan : @drawable/general_item_background_border}"`

Yes I think tablet and mobile layout can't be merged further. LGTM

**Review by bkaur-bkj — APPROVED** [review]
@rt4914 PTAL, It LGTM

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -605,4 +605,8 @@
   <dimen name="topic_lessons_title_margin_start">32dp</dimen>
   <dimen name="topic_lessons_title_margin_top">32dp</dimen>
   <dimen name="topic_lessons_title_margin_end">32dp</dimen>
+
+  <!--OptionAudioLanguage-->
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml:609]: Nit: `<!-- OptionAudioLanguage -->`

File: app/src/main/res/values-land/dimens.xml [hunk 2]
```diff
@@ -408,4 +408,8 @@
   <!-- TopicLessonsTitle -->
   <dimen name="topic_lessons_title_margin_start">100dp</dimen>
   <dimen name="topic_lessons_title_margin_end">100dp</dimen>
+
+  <!--OptionAudioLanguage-->
```
Comment (rt4914) [inline app/src/main/res/values-land/dimens.xml:412]: Ditto

**Review by rt4914 — CHANGES_REQUESTED** [review]
Similar to this comment https://github.com/oppia/oppia-android/pull/4125#pullrequestreview-862412352

Even all files in this PR can be merged. Also please recreate the after screenshots once these changes are done.

**Pull request closed without merging**